readthedocs/templates/errors/404/include_error_message.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+<section>
+  <p>
+    {% blocktrans trimmed with project_slug=project.slug|default:_("unknown slug") %}
```

Comment:
We should always assume to have a project/project slug here (all the templates that include this file should have it available)

Reply:
I removed the `|default` filter if that is what you meant here.